test/e2e/suite/issuers/acme/certificate/http01.go

```diff
@@ -222,4 +226,49 @@ var _ = framework.CertManagerDescribe("ACME Certificate (HTTP01)", func() {
 		err = h.WaitCertificateIssuedValid(f.Namespace.Name, certificateName, time.Minute*5)
 		Expect(err).NotTo(HaveOccurred())
 	})
+
+	It("should obtain a signed certificate with a single CN from the ACME server after solver pod killed", func() {
```

Comment:
maybe change to "should automatically recreate challenge pod and still obtain a certificate if it is manually deleted"? (just thinking about how we'll read these test failure messages in a few months time when we start breaking things :sweat_smile:)